packages/yuudachi/locales/en-US/translation.json

```diff
@@ -124,25 +125,59 @@
 			},
 			"anti_raid_nuke": {
 				"errors": {
+					"invalid_file_type": "Invalid file type provided, must be a `.txt` or `.json` file.",
+					"invalid_avatar": "Invalid avatar format, must be an raw hash, a userId, a URL or `none`.",
```

Comment:
```suggestion
					"invalid_avatar": "Invalid avatar format, must be an raw hash, a user id, a URL or `none`.",
```

Reply:
Changed in [801e5f6](https://github.com/Naval-Base/yuudachi/pull/1069/commits/801e5f629bf9e6249b6942ce8c7c3a09b006acbb)